beamer/contracts.py

```diff
@@ -35,7 +35,7 @@ def _load_entry(self, name: str) -> _CacheEntry:
         with path.open("rt") as f:
             data = json.load(f)
         return ABIManager._CacheEntry(
-            abi=data["abi"], bytecode=data["runtimeBytecode"]["bytecode"]
+            abi=data["abi"], bytecode=data["deploymentBytecode"]["bytecode"]
```

Comment:
github sucks so I will comment everything in here:
```
    _CacheEntry = namedtuple("_CacheEntry", ("abi", "bytecode"))
```
I think we should rename `bytecode` field of `_CacheEntry` to `deployment_bytecode` to reflect what it really is.
Someone could otherwise mistake it for runtime bytecode.

Similarly, I would rename
```
    def get_bytecode(self, name: str) -> str:
```
to
```
    def get_deployment_bytecode(self, name: str) -> str:
```
to be consistent.